# Parameters
Lfw: 2.5 #default = 0.8 forward look ahead distance (m)
goal_radius: 0.5 # unit: m
controller_freq: 20 # unit: Hz

pathLength: 10.0

odom_topic: "odom"
plan_topic: "move_base/GlobalPlanner/plan"
goal_topic: "move_base_simple/goal"
amcl_topic: "amcl_pose"

map_frame: "map"
odom_frame: "odom"

lah_point_topic: "lookAhead_point"
lah_marker_topic: "LookAhead/path_marker"
